scripts/common.sh

```diff
@@ -52,6 +52,14 @@ export INSIGHT_TARBALL_TYPE=`get_make_var insight_tarball_type`
 export ARM_BINUTILS_TARBALL_TYPE=`get_make_var arm_binutils_tarball_type`
 export ARM_GCC_TARBALL_TYPE=`get_make_var arm_gcc_tarball_type`
 
+export SH_BINUTILS_URL=ftp.gnu.org/gnu/binutils/binutils-${SH_BINUTILS_VER}.tar.${SH_BINUTILS_TARBALL_TYPE}
```

Comment:
Thank you! 👍 
You are right, it's much better to handle URLs like other variables instead of computing them in the `download.sh` script.